### PR TITLE
Add more MDM agent UnitTests

### PR DESCRIPTION
**Pull request by joenpera**
- New UnitTests (SECO-4685).
- psutil dependency removed (SECO-4863) to help execution in a target device
- comms_status test fixes (SECO-4874)

Jira-Id: SECO-4685, SECO-4863, SECO-4874

### CODE REVIEW

**Review by pentestiing — APPROVED** [review]
Great job! :clap:

File: modules/sc-mesh-secure-deployment/src/nats/tests/test_command.py [hunk 1]
```diff
@@ -1,16 +1,57 @@
...
+        # msg = '{"api_version": 1, "cmd": "APPLY", "radio_index":' + f"{index}" + '}'
+        # ret, _, _ = self.command.handle_command(msg, cc)
+        # self.assertEqual(ret, "FAIL")
+        #
+        # msg = ('{ "api_version": 1, "role": "sleeve", "radios": [ {"radio_index": "0", '
+        #        '"ssid": "test_mesh2", "key": "1234567890", "country": "US", "frequency": "2412", '
+        #        '"frequency_mcc": "2412", "mptcp": "disable", "priority": "high_throughput", '
+        #        '"tx_power": "15", "mode": "mesh", "mesh_vif": "wlp2s0", "slaac": ""}]}')
+        # ret, _ = self.settings.handle_mesh_settings(msg, "/tmp/")
+        # self.assertEqual(ret, "OK")
```
[the remark this answers is not in the document]
Comment (joenpera) [reply modules/sc-mesh-secure-deployment/src/nats/tests/test_command.py:54]: Future work to cover all interfaces from comms_command and therefor add TODO.

[the remark this answers is not in the document]
Comment (joenpera) [reply modules/sc-mesh-secure-deployment/src/nats/tests/test_command.py:54]: I will create backlog item from this one

File: modules/sc-mesh-secure-deployment/src/nats/tests/service_discovery_helper.py [hunk 2]
```diff
@@ -1,37 +1,21 @@
...
 
 
 SERVICE_TYPE: str = '_mdm._tcp.local.'
 SERVICE_NAME: str = 'MDM Service'
 HOSTNAME: str = 'defaultmdm.local'
 
-IPV4_ANY_ADDR: str = '127.0.0.1'
-IPV6_ANY_ADDR: str = '::'
+IPV4_ANY_ADDR: bytes = b'\x7f\x00\x00\x01'
+IPV6_ANY_ADDR: bytes = b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x01'
```
[the remark this answers is not in the document]
Comment (joenpera) [reply modules/sc-mesh-secure-deployment/src/nats/tests/service_discovery_helper.py]: [df035b4](https://github.com/tiiuae/mesh_com/pull/439/commits/df035b41931cd3f78cf7bf64d2d9940fa164b681)

**Pull request merged**